src/app/browse-by/browse-by-metadata-page/browse-by-metadata-page.component.ts

```diff
@@ -54,7 +55,7 @@ export class BrowseByMetadataPageComponent implements OnInit {
   paginationConfig: PaginationComponentOptions = Object.assign(new PaginationComponentOptions(), {
     id: BBM_PAGINATION_ID,
     currentPage: 1,
-    pageSize: 20
+    pageSize: environment.browseBy.pageSize,
```

Comment:
It's better to inject the environment using
```typescript
@Inject(APP_CONFIG) protected appConfig: AppConfig
```
in the constructor, and then use `this.appConfig.browseBy.pageSize` later on. That makes it easier for us to mock the environment in tests.